crates/pallet-domains/src/lib.rs

```diff
@@ -1555,7 +1607,13 @@ impl<T: Config> Pallet<T> {
             .extrinsics
             .iter()
             .fold(0, |acc, xt| acc + xt.encoded_size() as u32);
+
         ensure!(max_size >= bundle_size, BundleError::BundleTooLarge);
+        ensure!(
+            opaque_bundle.size() == bundle_size,
```

Comment:
Don't the `opaque_budnlde.size()` includes the ER as well ?

Reply:
I don't think so according to the [spec](https://www.notion.so/subspacelabs/Domains-v2-Specification-3fb0ec6e4d204c4881a7df50ef58da8f?pvs=4#37dc09fa7c604b04a0e79b67adf9c405) and the comment fo this field:
https://github.com/subspace/subspace/blob/6b5ef44bcb7ceb9e2076795c1092cdb19d13b972/crates/sp-domains/src/lib.rs#L210-L213

Reply:
hmmm. @dariolina dont we want to also include ER size here as well as this is as well stored

Reply:
So far we have been only talking about the body storage cost, of extrinsics inside it. Currently, yeah, the operators put ERs in basically for free. In this way they are similar to block headers: those are also archived and stored but nobody "paid" for them.
If we make operators pay for ERs storage too we will have to charge users more then. 
I'll bring up this question in R&D, but IMO, it's fine as is.

Reply:
We've discussed this in R&D and the consensus was that ERs should pay, but we will open a separate issue for it to define design and spec. This PR can merge with just the bundle body paying.